dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/LoggerController.java

```diff
@@ -102,6 +99,7 @@
     @GetMapping(value = "/download-log")
     @ResponseBody
     @ApiException(DOWNLOAD_TASK_INSTANCE_LOG_FILE_ERROR)
+    @AccessLogAnnotation
     public ResponseEntity downloadTaskLog(@ApiIgnore @RequestAttribute(value = Constants.SESSION_USER) User loginUser,
                                           @RequestParam(value = "taskInstanceId") int taskInstanceId) {
         byte[] logBytes = loggerService.getLogBytes(taskInstanceId);
```

Comment:
There is no need to record the response of this method, it will print the byte array.